src/lib/util/ModUtil.php

```diff
@@ -296,6 +306,9 @@ public static function setVar($modname, $name, $value = '')
 
     /**
      * The setVars method sets multiple module variables.
+     * @deprecated at Core-1.4.1
+     * @see \Zikula\ExtensionsModule\Api\VariableApi::setAll()
```

Comment:
Whoops, you are right. Apparently, `@use` is the evil one: https://github.com/zikula/core/pull/1517/files#diff-aca64025345ee21fbaa0d4436006a52fL266